week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -38,7 +65,16 @@ Write a function that:
 */
 
 function formatPercentage(arr) {
-  
+  for (var i = 0; i < arr.length; i++) {
+    if (arr[i] > 100) {
+      arr[i] = 100;
+    }
+    if (!Number.isInteger(arr[i])) {
+      arr[i] = arr[i].toFixed(2);
+    }
+    arr[i] = `${arr[i]}%`;
+  }
+  return arr;
```

Comment:
Looks great.

What would happen if we didn't have that `if (!Number.isInteger(arr[i]))`?

Also, can you think of a way to do the same returning a new array instead of modifying `arr`?